pixart.js

```diff
@@ -1 +1,31 @@
 
+$(document).ready(function(){
+
+  var button = $("button");
+  var input = $("input");
+
+
+
+  button.on("click", changeBrushColor);
+  button.on("keypress", changeBrushColor);
+
+
+  function changeBrushColor(evt) {
+    evt.preventDefault();
+
+
+    var newColor = input.val();
+    $(".brush").css("background", newColor);
+  }
+
+
+  for ( var i = 0; i < 8000; i++) {
+    $("body").append("<div class='square'></div>");
+  };
+  $(".square").mouseover(function(evt) {
+    evt.preventDefault();
+    $(this).css("background", ($('input#color-field').val()));
```

Comment:
i would consider using the brush box color rather than the input values. Otherwise, whats the point of setting the brush color in the other event listener.